controls/V-40792.rb

```diff
@@ -18,29 +20,32 @@
 transmission, the session were to be hijacked, the user's credentials would not
 be compromised.
   "
+  desc  "rationale", ""
+  desc  "check", "
+  Review the NGINX web server documentation and configuration to determine if
+  server-side session management is configured.
+
+  If it is determined that the web server is not required to perform session management, this check is Not Applicable. 
+
+  If it is not configured, this is a finding.
+  "
+  desc  "fix", "Configure the NGINX web server to perform server-side session
+management."
   impact 0.5
+  tag "severity": "medium"
   tag "gtitle": "SRG-APP-000001-WSR-000002"
   tag "gid": "V-40792"
   tag "rid": "SV-53023r3_rule"
   tag "stig_id": "SRG-APP-000001-WSR-000002"
   tag "fix_id": "F-45949r2_fix"
   tag "cci": ["CCI-000054"]
   tag "nist": ["AC-10", "Rev_4"]
-  tag "false_negatives": nil
-  tag "false_positives": nil
-  tag "documentable": false
-  tag "mitigations": nil
-  tag "severity_override_guidance": false
-  tag "potential_impacts": nil
-  tag "third_party_tools": nil
-  tag "mitigation_controls": nil
-  tag "responsibility": nil
-  tag "ia_controls": nil
-  tag "check": "Review the web server documentation and configuration to
-determine if server-side session management is configured.
 
-If it is not configured, this is a finding."
-  tag "fix": "Configure the web server to perform server-side session
-management."
+  describe "Manual Check" do
```

Comment:
This should be more descriptive.. a manual check of what ... does it communicate what the user needs to do and check? Perhaps something like:
`This test requires a Manual Review` ...
We should standardize this lang across all the controls